src/views/profile/bookmarks-view/index.tsx

```diff
@@ -24,12 +24,11 @@ const ProfileBookmarksView = () => {
  * The content of the ProfileView that is gated behind authentication.
  */
 const ProfileBookmarksViewContent = () => {
-	const { bookmarks, isFetching, isRefetching } = useAllBookmarks({
+	const { bookmarks, isLoading } = useAllBookmarks({
 		enabled: true,
 	})
-	const isFirstLoad = isFetching && !isRefetching
 
-	if (isFirstLoad) {
+	if (isLoading) {
```

Comment:
Since we know that this view will only render in an authenticated state, adjust this to use `isLoading` which is [explained as](https://react-query-v3.tanstack.com/guides/queries#query-basics): 
<img width="741" alt="Screen Shot 2022-08-24 at 11 47 17 AM" src="https://user-images.githubusercontent.com/36613477/186498887-5e5b456d-cca9-4a11-965a-f51b625b08d3.png">

Seems like what we want in this case.